packages/web-components/src/components/content-section/content-section.ts

```diff
@@ -0,0 +1,139 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { customElement, html, property, LitElement } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+import settings from 'carbon-components/es/globals/js/settings';
+import ddsSettings from '@carbon/ibmdotcom-utilities/es/utilities/settings/settings.js';
+import styles from './content-section.scss';
+
+const { prefix } = settings;
+const { stablePrefix: ddsPrefix } = ddsSettings;
+
+/**
+ * Content Section theme
+ */
+export enum CONTENT_SECTION_THEME {
+  /**
+   * Carbon White theme (default)
+   */
+  WHITE = 'white',
+
+  /**
+   * Carbon Gray 90 theme
+   */
+
+  G10 = 'g10',
+
+  /**
+   * Carbon Gray 90 theme
+   */
+  G90 = 'g90',
+
+  /**
+   *  Carbon Gray 100 theme
+   */
+  G100 = 'g100',
+}
+
+/**
+ * The Content Section component for use with cardSection
+ *
+ * @element dds-content-section
+ */
+@customElement(`${ddsPrefix}-content-section`)
+class DDSContentSection extends LitElement {
+  /**
+   * Returns a class-name based on the type parameter type
+   */
+  protected _getTypeClass() {
+    return classMap({
+      [`${this.customClassName}`]: this.customClassName,
+      [`${prefix}--content-section--${this.theme}`]: true,
+      [`${prefix}--content-section`]: true,
+    });
+  }
+
+  /**
+   * Renders Content Section heading slot
+   */
+  protected _renderHeading() {
+    const { heading } = this;
+    return html`
+      <slot name="heading">${heading}</slot>
+    `;
+  }
+
+  /**
+   * Renders children
+   */
+  protected __renderChildren() {
+    const { children } = this;
+    return html`
+      <slot name="children">${children}</slot>
```

Comment:
Web Components has no notion like React's `children` property. The closest to React `children` property is the default `<slot>`:

```suggestion
      <slot></slot>
```